Panic during searchv2 with one SN down

```
2026-04-28T15:44:32.360-0400	info	cache/clients.go:192	initializing connection to the SN...	{"scope": "read", "public key": "03fe344aa2409866351c1536cec910528c59f9c2af08f60c62a2e2d22fa7b20c11", "address": "/dns4/localhost/tcp/53967"}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x18 pc=0x105064a1c]

goroutine 360 [running]:
github.com/nspcc-dev/neofs-node/pkg/services/object.(*Server).ProcessSearch(0x140002288f0, {0x1056a0130, 0x14000ec8150}, 0x1400076d900)
	github.com/nspcc-dev/neofs-node/pkg/services/object/server.go:2152 +0xc1c
github.com/nspcc-dev/neofs-node/pkg/services/object.(*Server).processSearchRequest(0x140002288f0, {0x1056a0130, 0x14000ec8150}, 0x1400076d900)
	github.com/nspcc-dev/neofs-node/pkg/services/object/server.go:2014 +0x52c
github.com/nspcc-dev/neofs-node/pkg/services/object.(*Server).SearchV2(0x140002288f0, {0x1056a0130, 0x14000ec8150}, 0x1400076d900)
	github.com/nspcc-dev/neofs-node/pkg/services/object/server.go:1951 +0x314
github.com/nspcc-dev/neofs-sdk-go/proto/object._ObjectService_SearchV2_Handler({0x10565b340, 0x140002288f0}, {0x1056a0130, 0x14000ec8150}, 0x14000e80500, 0x0)
	github.com/nspcc-dev/neofs-sdk-go@v1.0.0-rc.18.0.20260417090255-d6b86c01a5af/proto/object/service_grpc.pb.go:784 +0x1b8
google.golang.org/grpc.(*Server).processUnaryRPC(0x140000fe248, {0x1056a0130, 0x14000ec80c0}, 0x14000ea8340, 0x1400034e270, 0x1400012ea30, 0x0)
	google.golang.org/grpc@v1.79.3/server.go:1429 +0xc94
google.golang.org/grpc.(*Server).handleStream(0x140000fe248, {0x1056a0ad8, 0x14000313860}, 0x14000ea8340)
	google.golang.org/grpc@v1.79.3/server.go:1855 +0x89c
google.golang.org/grpc.(*Server).serveStreams.func2.1()
	google.golang.org/grpc@v1.79.3/server.go:1064 +0x74
created by google.golang.org/grpc.(*Server).serveStreams.func2 in goroutine 372
	google.golang.org/grpc@v1.79.3/server.go:1075 +0x120
```

--- a/pkg/services/object/server.go
+++ b/pkg/services/object/server.go
@@ -2149,7 +2149,7 @@ func (s *Server) ProcessSearch(ctx context.Context, req *protoobject.SearchV2Req
 			}
 			if searchRes.err != nil {
 				var inc = new(apistatus.Incomplete)
-				inc.SetMessage(fmt.Sprintf("last error: %s", err.Error()))
+				inc.SetMessage(fmt.Sprintf("last error: %s", searchRes.err.Error()))
 				incomplete = inc
 				continue
 			}

